.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
nit: Remove.

We should add this to all our .gitignore files across repos

Reply:
Actually, this was very intentional.  We already have some VS code settings files committed to source, both in Universe https://github.com/aspnet/Universe/tree/dev/.vscode and in some runtime repos too https://github.com/aspnet/KestrelHttpServer/tree/dev/.vscode

Reply:
I think those got there as an accident. See https://github.com/aspnet/Identity/issues/1428
That said, I don't really care about it. The only problem I see is that those are User specific environment files, so they will change if a user changes something